<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReceiveCoinsDialog</class>
 <widget class="QWidget" name="ReceiveCoinsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>420</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="receiveCoinsLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="receiveCoinsLeftSide">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1">
      <item>
       <widget class="QFrame" name="requestPaymentsForm">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="7" column="2">
            <widget class="QCheckBox" name="reuseAddress">
             <property name="text">
              <string>R&amp;euse an existing receiving address (not recommended)</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>&amp;Message:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="buddy">
              <cstring>reqMessage</cstring>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QLineEdit" name="reqLabel">
            </widget>
           </item>
           <item row="6" column="2">
            <widget class="QLineEdit" name="reqMessage">
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Use this form to request payments. All fields are &lt;b&gt;optional&lt;/b&gt;</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>&amp;Label:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="buddy">
              <cstring>reqLabel</cstring>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>&amp;Amount:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="buddy">
              <cstring>reqAmount</cstring>
             </property>
            </widget>
           </item>
           <item row="5" column="2">
            <widget class="CoinAmountField" name="reqAmount">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="8" column="2">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="receiveButton">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>&amp;Request payment</string>
               </property>
               <property name="icon">
                <iconset resource="../dogecoin.qrc">
                 <normaloff>:/icons/receiving_addresses</normaloff>:/icons/receiving_addresses</iconset>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="clearButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Clear</string>
               </property>
               <property name="icon">
                <iconset resource="../dogecoin.qrc">
                 <normaloff>:/icons/remove</normaloff>:/icons/remove</iconset>
               </property>
               <property name="autoDefault">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>1</width>
          <height>8</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="buttonsForPaymentsHistory">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="paymentsHistoryLabel">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Requested payments history</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableView" name="recentRequestsView">
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <property name="tabKeyNavigation">
            <bool>false</bool>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="recentViewButtons_horizontalLayout">
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="removeRequestButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Remove the selected entries from the list</string>
             </property>
             <property name="text">
              <string>Remove</string>
             </property>
             <property name="icon">
              <iconset resource="../dogecoin.qrc">
               <normaloff>:/icons/remove</normaloff>:/icons/remove</iconset>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="receiveRequestFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>420</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="receiveRequest_verticalLayout">
      <item>
       <widget class="QRImageWidget" name="paymentRequestQRCode">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>320</height>
         </size>
        </property>
        <property name="toolTip">
         <string>QR Code</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="belowQR_vSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>1</width>
          <height>4</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QTextEdit" name="outUri">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="tabChangesFocus">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutRightSide">
        <item>
         <widget class="QPushButton" name="btnCopyURI">
          <property name="text">
           <string>Copy &amp;URI</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCopyAddress">
          <property name="text">
           <string>Copy &amp;Address</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSaveAs">
          <property name="text">
           <string>&amp;Save Image...</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CoinAmountField</class>
   <extends>QLineEdit</extends>
   <header>coinamountfield.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QRImageWidget</class>
   <extends>QLabel</extends>
   <header>qrimagewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../dogecoin.qrc"/>
 </resources>
 <connections/>
</ui>
